pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -819,6 +806,47 @@ func (r *ReconcileHyperConverged) detectTaintedConfiguration(req *common.HcoRequ
 	}
 }
 
+func (r *ReconcileHyperConverged) setOperatorConditionUpgradable(ctx context.Context, status metav1.ConditionStatus) error {
+	cond := metav1.Condition{
+		Type:    olmUpgradableConditionType,
+		Status:  metav1.ConditionTrue,
+		Reason:  reconcileCompleted,
+		Message: reconcileCompletedMessage,
```

Comment:
Not sure we need any Reason and mesaage here. I would keep it empty. Anyway, "reconcile completed" is not the right message.